docker-compose.override.test.yml

```diff
@@ -0,0 +1,33 @@
+version: '3.7'
+
+services:
+
+  app:
+    environment:
+      # run flush, migrate and collectstatic in entrypoint
```

Comment:
I think that comment is wrong (the entrypoint doesn't do that), yet it's also in `docker-compose.override.local.yml`. Should we reintroduce that behaviour, or remove these comments ?

Reply:
I removed the comments.